src/test/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidationTest.java

```diff
@@ -211,14 +212,27 @@ void validateDoesNotCallOfficerServiceWhenOfficerValidationFeatureFlagFalse() th
     }
 
     @Test
-    void validateDoesThrowOnCompanyWithZeroOfficers() throws ServiceException {
+    void validateThrowsExceptionOnCompanyWithZeroOfficers() throws ServiceException {
         companyOfficerValidation = new CompanyOfficerValidation(officerService,true, false);
-        when(officerService.getOfficers(COMPANY_NUMBER)).thenReturn(new OfficersApi());
+        OfficersApi officersApi = new OfficersApi();
+        officersApi.setItems(Collections.emptyList());
+        officersApi.setActiveCount(0L);
+        when(officerService.getOfficers(COMPANY_NUMBER)).thenReturn(officersApi);
         var result = companyOfficerValidation.isOfficerDirector(mockOfficers.getItems(), mockOfficers.getActiveCount());
         var ex = assertThrows(EligibilityException.class, () -> companyOfficerValidation.validate(companyProfileApi));
 
         assertEquals(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS,ex.getEligibilityStatusCode() );
         assertFalse(result);
     }
 
+    @Test
+    void validateThrowsExceptionOnCompanyWithNullOfficerList() throws ServiceException {
+        companyOfficerValidation = new CompanyOfficerValidation(officerService,true, true);
+        when(officerService.getOfficers(COMPANY_NUMBER)).thenReturn(new OfficersApi());
```

Comment:
can you set the active count to null on the new OfficersApi, so that we don't rely on the null value being set by new OfficersApi. Someone might change OfficersApi.activeCount in future to a primitive or even set a default value and the test would fail when it should still pass.

Reply:
Yes, good point, but it's the list that should really be set to null. I'll set that and the count to null so all bases are covered.